ml/src/main/scala/org/apache/spark/ml/h2o/algos/H2OAutoML.scala

```diff
@@ -107,9 +107,12 @@ class H2OAutoML(override val uid: String) extends Estimator[H2OMOJOModel]
         " your 'excludeAlgo', 'maxModels' or 'maxRuntimeSecs' properties.") with NoStackTrace
     }
 
-    val mojoData = ModelSerializationSupport.getMojoData(aml.leader())
+    val binaryModel = aml.leader()
+    val mojoData = ModelSerializationSupport.getMojoData(binaryModel)
     val model = H2OMOJOModel.createFromMojo(mojoData, Identifiable.randomUID(aml.leader()._parms.algoName()))
-    model.setConvertUnknownCategoricalLevelsToNa(true)
+
+    // pass some parameters set on algo to model
+    model.setConvertUnknownCategoricalLevelsToNa(getConvertUnknownCategoricalLevelsToNa())
```

Comment:
Would it be possible to use ```copyValues``` for copying parameter values to the trained models or are there any limitations?

Reply:
Good idea, let me try it

Reply:
It required that the default values are the same in the original and target param instance which right now is not true. I would iterate towards it, but right now we can't afford it